packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -94,6 +94,10 @@ export class RouterliciousRestWrapper extends RestWrapper {
                     .catch(reject);
             }, responseBody.retryAfter * 1000));
         }
+        if (response.status === 502 && canRetry) {
```

Comment:
What is the reason for 502?  Should we have more generic way for retry?  @znewton

Reply:
I spent a large amount of time trying to pin down the reason but didn't make much headway. It showed up very infrequently in my CI runs and I'm unsure if this retry will actually even pass the tests when it occurs. I'm definitely open to a better solution here

Reply:
There is one case where our service generates the 502 code (`server\routerlicious\packages\services-client\src\restWrapper.ts`:170)

I wonder whether we can narrow it down if that is the case by changing that error code and see if it repros with the new error code?   Or check the tinylicious log and see if can find it there?

How often does this happen?

Reply:
From my CI runs, it only occurred 1/20 builds. Let me try changing that error code and running the tests locally a while to see if I can reproduce it and narrow it down

Reply:
Spent a good amount of time on this Friday and wasn't able to repro it locally with the error code changed